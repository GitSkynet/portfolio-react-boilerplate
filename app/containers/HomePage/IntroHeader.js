import styled from 'styled-components';

const IntroHeader = styled.header`
  height: 100%;
  width: 100vmin;
  z-index: 9;
`;

export default IntroHeader;
