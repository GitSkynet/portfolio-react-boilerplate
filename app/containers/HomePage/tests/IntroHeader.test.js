import React from 'react';
import { shallow } from 'enzyme';

import IntroHeader from '../IntroHeader';

describe('<IntroHeader />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<IntroHeader />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
