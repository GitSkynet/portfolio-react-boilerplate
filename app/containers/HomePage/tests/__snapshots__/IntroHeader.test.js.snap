// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroHeader /> should render and match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IntroHeader__IntroHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "IntroHeader__IntroHeader-dYiXvG fHMnF",
    },
    "ref": null,
    "rendered": null,
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "IntroHeader__IntroHeader-dYiXvG fHMnF",
      },
      "ref": null,
      "rendered": null,
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
